src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -85,6 +86,16 @@ private static void getLocationsFromPath(List<String> possiblePaths) {
         if (path.endsWith("google-cloud-sdk" + File.separator + "bin")) {
           possiblePaths.add(path.substring(0, path.length() - 4));
         }
+
+        Path possibleLink = Paths.get(path, "gcloud");
+        if (Files.exists(possibleLink) && Files.isSymbolicLink(possibleLink)) {
+          try {
+            Path destination = Files.readSymbolicLink(possibleLink);
```

Comment:
- Following a symlink seems a nice idea, but why once? Maybe we should iterate until it is not a symlink.
- But then, if the SDK is updated in the future in a way that `gcloud` becomes a symlink and the actual binary is placed in some weird sub-folder, it might break. (Maybe this is stretched too far.)
- Might be a good idea to check existence of `destination`?